database/guests.ts

```diff
@@ -1,44 +1,55 @@
-declare module globalThis {
-  let guestsDatabase: Guest[];
-}
-
-// Initialize database to empty array only once
-//
-// TODO: Add a full PostgreSQL database connection
-if (!('guestsDatabase' in globalThis)) {
-  globalThis.guestsDatabase = [];
-}
-
-const guests = globalThis.guestsDatabase;
-
-console.log('guests in db', guests);
-
-export type Guest = {
-  id: number;
-  firstName: string;
-  lastName: string;
-  attending: boolean;
-};
-
-export function getGuests() {
+import { Guest } from '../migrations/00000-createTableGuests';
+import { sql } from './connect';
+
+export const getGuests = async () => {
```

Comment:
`const ... = async () =>` arrow function style used here to be closer to the `cache()` database query function code we have in the Next.js example:

https://github.com/upleveled/next-js-example-spring-2024-atvie/blob/869d615738ee991fd80f31b44dfa3c4f79a7ce3d/database/animals.ts#L50-L62